org.eclipse.xtext.tests/src/org/eclipse/xtext/xtext/Issue1161Test.xtend

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Mälardalen University (http://www.mdh.se) and others.
```

Comment:
Could you avoid the Umlaut `ä` here? Although encoding is usually set appropriate it might lead to problems when encoding is not explicitly set.

Reply:
Yes, sorry